src/holidays/fr.yaml

```diff
@@ -972,34 +1176,59 @@ Martinique:
       '2021': [2, 7, 2, 21]
       '2022': [2, 19, 3, 6]
       '2023': [2, 11, 2, 26]
+      '2024': []
+      '2025': [2, 22, 3, 9]
+      '2026': [2, 7, 2, 22]
     - name: Vacances de Pâques
       '2018': [3, 25, 4, 8]
       '2019': [4, 14, 4, 28]
       '2020': [4, 5, 4, 19]
       '2021': [3, 28, 4, 11]
       '2022': [4, 9, 4, 24]
       '2023': [4, 1, 4, 16]
+      '2024': []
+      '2025': [4, 12, 4, 27]
+      '2026': [3, 28, 4, 12]
     - name: Pont de l'Ascension
-      '2018': [5, 8, 5, 13]
+      '2018': [5, 10, 5, 10]
       '2019': [5, 30, 6, 2]
-      '2020': [5, 14, 5, 15]
-      '2022': [5, 27, 5, 27]
+      '2020': [5, 21, 5, 24]
+      '2021': [5, 14, 5, 16]
+      '2022': [5, 26, 5, 27]
       '2023': [5, 17, 5, 21]
+      '2024': [5, 8, 5, 12] # double check
```

Comment:
If I interpret the following source correctly, the values are correct:
https://vacances-scolaires-education.fr/ascension-2024.html

Please remove the comment.